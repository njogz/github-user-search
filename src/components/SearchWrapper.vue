<template>
  <v-container>
    <v-row>
      <v-col>
        Search more than 40M Github users
      </v-col>
    </v-row>
    <search-box />
    <search-results />
  </v-container>
</template>

<script>
import SearchBox from '@/components/SearchBox.vue';
import SearchResults from '@/components/SearchResults.vue';

export default {
  name: 'SearchWrapper',
  components: {
    SearchBox,
    SearchResults,
  },
  data: () => ({
  }),
};
</script>
