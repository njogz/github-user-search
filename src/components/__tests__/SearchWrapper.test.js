import { shallowMount, createLocalVue } from '@vue/test-utils';
import SearchWrapper from '@/components/SearchWrapper.vue';
import Vuetify from 'vuetify';

const localVue = createLocalVue();
localVue.use(Vuetify);

describe('SearchWrapper.vue', () => {
  let vuetify;

  beforeEach(() => {
    vuetify = new Vuetify();
  });

  it('Should match the saved snapshot', () => {
    const wrapper = shallowMount(SearchWrapper, {
      localVue,
      vuetify,
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
