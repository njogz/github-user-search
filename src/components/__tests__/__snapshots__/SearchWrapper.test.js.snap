// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchWrapper.vue Should match the saved snapshot 1`] = `
<v-container-stub tag="div">
  <v-row-stub tag="div">
    <v-col-stub tag="div">
      Search more than 40M Github users
    </v-col-stub>
  </v-row-stub>
  <search-box-stub></search-box-stub>
  <search-results-stub></search-results-stub>
</v-container-stub>
`;
